tests/acceptance/01_vars/02_functions/findlocalusers.cf

```diff
@@ -0,0 +1,74 @@
+body common control 
+{
+  inputs => { "../../default.cf.sub" };
+  bundlesequence => { default("$(this.promise_filename)") };
+  version => "1.0";
+}
+bundle agent init
+{
+  files:
+    "/tmp/test_folder1/."
+      create => "true";
+
+    "/tmp/test_folder2/."
+      create => "true";
+
+  users:
+      "test_user1"
+        policy => "present",
+        home_dir => "/tmp/test_folder1",
+        description => "TestUser 1",
+        group_primary => "users",
+        shell => "/bin/sh",
+        uid => "12345";
+
+      "test_user2"
+        policy => "present",
+        home_dir => "/tmp/test_folder2",
+        description => "TestUser 2",
+        group_primary => "users",
+        shell => "/bin/sh",
+        uid => "54321";
```

Comment:
This looks like an "unsafe" test. If I run the CFEngine acceptance tests, I don't want them to start editing `/etc/passwd` and adding users on my system. For those kinds of tests which make significant changes to the system they run on, we put them inside of a subdirectory called `./unsafe/` and you have to run the tests with `UNSAFE_TESTS=1` to trigger them.

This is also why GitHub Actions is failing currently.

After moving this test, maybe you can make a shorter safe test which just tries to call the function with uid 0, or something. You won't be able to assert so much about the result, but it will be a lot less intrusive.